modules/op-adding-triggers.adoc

```diff
@@ -161,8 +165,54 @@ Alternatively, you can create the `EvenListener` resource directly from the `pip
 ----
 $ oc create -f https://raw.githubusercontent.com/openshift/pipelines-tutorial/{pipelines-ver}/03_triggers/04_event_listener.yaml
 ----
-
-. Expose the `EventListener` service as an {product-title} route to make it publicly accessible:
+.. Add a label to enable the HTTPS (secure HTTP) connection to the `Eventlistener` resource:
++
+[source,terminal]
+----
+$ oc label namespace <ns-name> operator.tekton.dev/enable-annotation=enabled
+----
```

Comment:
This is the first step

I mean we add label first and then create eventlistener yaml

Reply:
Done. Thanks for the correction :)